### ISSUE TITLE
Dead project

### ISSUE THREAD

**Issue by NetOpWibby** [entropic-dev/entropic#385]
Is there an active fork?

I came across this project a few years ago and it was super lively but brand-spanking new. It's obviously a ghost town now but I do think an alternative to npm and perhaps a competitor to Verdaccio is worthwhile.

**Comment by FiGOBLAC** [comment]
> Is there an active fork?
> 
> I came across this project a few years ago and it was super lively but brand-spanking new. It's obviously a ghost town now but I do think an alternative to npm and perhaps a competitor to Verdaccio is worthwhile.

Not exactly... see https://github.com/entropic-dev/entropic/issues/365#issuecomment-675635682

**Comment by NetOpWibby** [comment]
Ah, thanks!
